linux/meminfo.go

```diff
@@ -2,43 +2,96 @@ package linux
 
 import (
 	"io/ioutil"
+	"reflect"
 	"strconv"
 	"strings"
 )
 
-/*
 type MemInfo struct {
-	Total      uint64
-	Free       uint64
-	Buffers    uint64
-	Cached     uint64
-	SwapCached uint64
-	Active     uint64
-	Inactive   uint64
+	MemTotal          uint64 `json:"mem_total"`
+	MemFree           uint64 `json:"mem_free"`
+	MemAvailable      uint64 `json:"mem_available"`
+	Buffers           uint64 `json:"buffers"`
+	Cached            uint64 `json:"cached"`
+	SwapCached        uint64 `json:"swap_cached"`
+	Active            uint64 `json:"active"`
+	Inactive          uint64 `json:"inactive"`
+	ActiveAnon        uint64 `json:"active_anon" altname:"Active(anon)"`
+	InactiveAnon      uint64 `json:"inactive_anon" altname:"Inactive(anon)"`
+	ActiveFile        uint64 `json:"active_file" altname:"Active(file)"`
+	InactiveFile      uint64 `json:"inactive_file" altname:"Inactive(file)"`
```

Comment:
Used struct field tags with an `altname` key for the stats that aren't valid field names in Go.